src/node_api.cc

```diff
@@ -123,6 +187,22 @@ v8::Local<v8::Value> V8LocalValueFromJsValue(napi_value v) {
   return local;
 }
 
+napi_status V8NameFromPropertyDescriptor(napi_env env,
```

Comment:
Also, just noticed, if you go with `static inline` can you re-align the arguments here? ;)